qiskit/primitives/statevector_estimator.py

```diff
@@ -46,9 +69,11 @@ def _run(self, pubs: list[EstimatorPub]) -> PrimitiveResult[PubResult]:
         return PrimitiveResult([self._run_pub(pub) for pub in pubs])
 
     def _run_pub(self, pub: EstimatorPub) -> PubResult:
+        rng = _get_rng(self._seed)
```

Comment:
You can simplify it (Chris' comment https://github.com/Qiskit/qiskit/pull/11566#discussion_r1455813113)
```suggestion
        rng = np.random.default_rng(self._seed)
```

Reply:
Cool! Thanks